owltorrent-network/src/main/java/edu/rice/owltorrent/network/SocketConnector.java

```diff
@@ -50,7 +50,15 @@ private SocketConnector(
       StorageAdapter storageAdapter,
       MessageReader messageReader,
       Socket peerSocket) {
-    super(peer, manager, storageAdapter, messageReader);
+    super(peer, manager, messageReader);
```

Comment:
maybe change this to this(peer, manager, messageReader, peerSocket), and then delete line 55?

Reply:
Socket should not be a field in PeerConnector, right?